javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,39 @@
+     // All at once
+     function translateAllAtOnce(){
+        
+        const promise1 = moveElement(document.querySelector('li:nth-child(1)'), {x: 20, y: 300}) 
+    
+        const promise2 =  moveElement(document.querySelector('li:nth-child(2)'), {x: 400, y: 300})
+        
+        const promise3 =   moveElement(document.querySelector('li:nth-child(3)'), {x: 400, y: 20})
+        
+        return(promise1,promise2,promise3)
+        
+        .then(() => {
+            console.log('Element has been moved');
+        })
+     }
+     translateAllAtOnce()
+      // one by one 
+     function translateOneByOne(){
+        const promise1 = new Promise( resolve => {
+       setTimeout(() => resolve(moveElement(document.querySelector('li:nth-child(1)'), {x: 20, y: 300})), 1000)
+        })
+        .then(() => {
+            console.log(' Red element has been moved');
+        }) 
+        const promise2 = new Promise( resolve => {
+            setTimeout(() => resolve(moveElement(document.querySelector('li:nth-child(2)'), {x: 400, y: 300})), 3000)
+             })
+             .then(() => {
+                 console.log(' Blue element has been moved');
+             }) 
+             const promise3 = new Promise( resolve => {
+                setTimeout(() => resolve(moveElement(document.querySelector('li:nth-child(3)'), {x: 400, y: 20})), 5000)
+                 })
+                 .then(() => {
+                     console.log(' Green element has been moved');
+                 }) 
+                  
+     }
+     translateOneByOne()
```

Comment:
This one doesn't move any circles. ;)
Also, you declared `promise1`, `promise2` and `promise3` but you never used them.